File: jetpack_java/src/main/java/com/kunminx/jetpack_java/sample_one_more_thing/ui/state/OneMoreThingViewModel.java


package com.kunminx.jetpack_java.sample_one_more_thing.ui.state;

import androidx.databinding.ObservableField;
import androidx.lifecycle.ViewModel;

/**
 * Create by KunMinX at 2020/6/1
 */
public class OneMoreThingViewModel extends ViewModel {

    public final ObservableField<String> pageAssetPath = new ObservableField<>();

    {
        pageAssetPath.set("best_practice.html");
    }
}

File: jetpack_kotlin/src/main/java/com/flywith24/jetpack_kotlin/sample_one_more_thing/ui/state/OneMoreThingViewModel.kt
package com.flywith24.jetpack_kotlin.sample_one_more_thing.ui.state

import androidx.databinding.ObservableField
import androidx.lifecycle.ViewModel

/**
 * @author Flywith24
 * @date   2020/6/10
 * time   00:35
 * description
 */
class OneMoreThingViewModel : ViewModel() {
    val pageAssetPath = ObservableField("use_jetpack.html")
}